### DISCUSSION TITLE
Wireguard support

### DISCUSSION THREAD

**Discussion by jlamande** [bottlerocket-os/bottlerocket#1975] [General]
Hi,

Is support of [Wireguard](https://www.wireguard.com/) (embedded in Linux kernel 5.6+) already available or planned ?

A typical use case: [node to node transparent encryption](https://docs.cilium.io/en/v1.10/gettingstarted/encryption-wireguard/) using Cillium ?

**Comment by bcressey** [comment]
Wireguard is enabled in all the variants that use a 5.10 kernel, which at this point is everything except aws-k8s-1.18 and aws-k8s-1.19.

I haven't tried it with Cilium but I'd expect it to work.
